server/src/js/auth.js

```diff
@@ -73,7 +73,7 @@ router.get('/permissions', async function login(req, res, next) {
 router.post('/login', async function login(req, res, next) {
   try {
     //try to init, in case of first visit
-    await init();
+    // await init();
```

Comment:
If remove this, we will get some trouble when we deploy the system to a new environment without initial data in DB